proca/lib/proca/staffer/role.ex

```diff
@@ -25,8 +25,19 @@ defmodule Proca.Staffer.Role do
       :manage_campaigns,
       :manage_action_pages
     ],
+    campaigner: [
+      :change_org_users,
+      :change_org_settings,
+      :change_campaign_settings,
+      :manage_campaigns,
+      :manage_action_pages
+    ],
     coordinator: [
       :change_campaign_settings
+    ],
+    translator: [
+      :change_org_users,
```

Comment:
Hi,

I think change_campaign_settings, not manage_campaigns is the correct permission for translator.
request change1: let's remove change_org_users, it gives too much power


note for me: understand if coordinator is still used, I dont see its point